src/designer/elsa-workflows-studio/src/modules/credential-manager/models/secrets/sqlserver.activity.ts

```diff
@@ -0,0 +1,99 @@
+import { Secret } from "../secret.model";
```

Comment:
I think the filename is incorrect.

Reply:
In this case, file is hosting SecretModel interface, so I think it is correct (I'm guessing that you are referring to one of the previous issues, where word mod**a**l was mistyped :)).

Reply:
You're absolutely right, my mistake 👍🏻